sigh/lib/sigh/runner.rb

```diff
@@ -134,7 +134,9 @@ def create_profile!
       UI.important "Creating new provisioning profile for '#{Sigh.config[:app_identifier]}' with name '#{name}'"
```

Comment:
Should the platform be added to this description since a tvOS app could share the same app_identifier as an iOS app